core/src/html/layout.rs

```diff
@@ -462,20 +464,47 @@ impl<'a, 'gc> LayoutContext<'a, 'gc> {
         is_device_font: bool,
     ) -> Option<Font<'gc>> {
         let library = context.library.library_for_movie_mut(self.movie.clone());
+        let font_name = span.font.to_utf8_lossy();
 
-        // If this text field is set to use device fonts, fallback to using our embedded Noto Sans.
         // Note that the SWF can still contain a DefineFont tag with no glyphs/layout info in this case (see #451).
         // In an ideal world, device fonts would search for a matching font on the system and render it in some way.
-        if let Some(font) = library
-            .get_font_by_name(&span.font.to_utf8_lossy(), span.bold, span.italic)
-            .filter(|f| !is_device_font && f.has_glyphs())
-            .or_else(|| context.library.device_font())
-        {
-            self.font = Some(font);
-            return self.font;
+        if !is_device_font {
+            if let Some(font) = library
+                .get_font_by_name(&font_name, span.bold, span.italic)
+                .filter(|f| f.has_glyphs())
+            {
+                return Some(font);
+            }
+        }
+
+        if let Some(font) = context.library.load_device_font(
+            &font_name,
+            context.ui,
+            context.renderer,
+            context.gc_context,
+        ) {
+            return Some(font);
         }
 
-        None
+        // [NA] I suspect that there might be undocumented more aliases.
+        // I think I've seen translated versions of these used in the wild...
+        let default_font = match font_name.deref() {
+            "_serif" => DefaultFont::Serif,
+            "_typewriter" => DefaultFont::Serif,
```

Comment:
I see that DefaultFont has a `TypeWriter` variant, should that be used instead?

Reply:
Yep, thanks for catching it. Fixed!